### PR TITLE
issue #fix fixed side menu issue

### PR DESCRIPTION
**Pull request by GSuma21**

### CODE REVIEW

File: src/app/core/services/util/util.service.ts [hunk 1]
```diff
@@ -6,12 +6,18 @@ import { environment } from 'src/environments/environment';
...
 import { ModelComponent } from 'src/app/shared/components/model/model.component';
 import * as Bowser from "bowser"
+import { Subject } from 'rxjs/Subject';
 
 @Injectable({
   providedIn: 'root',
 })
 export class UtilService {
   modal: any;
+  public Subject = new Subject<boolean>();
```
Comment (kiranharidas187) [inline src/app/core/services/util/util.service.ts:16]: provide proper name for the subject

File: src/app/core/services/util/util.service.ts [hunk 2]
```diff
@@ -6,12 +6,18 @@ import { environment } from 'src/environments/environment';
...
+
+  passValue(data:boolean) {
```
Comment (kiranharidas187) [inline src/app/core/services/util/util.service.ts:18]: provide proper name

File: src/app/modules/admin/manage-list/manage-list.component.ts [hunk 3]
```diff
@@ -78,6 +81,23 @@ export class ManageListComponent implements OnInit {
       this.util.openModal(componenProps).then((data)=>{
       })
     }
+    async prefillData(requestDetails: any) {
```
Comment (kiranharidas187) [inline src/app/modules/admin/manage-list/manage-list.component.ts:84]: can this function be generic as it is used in multiple pages??

File: src/app/core/services/profile/profile.service.ts [hunk 4]
```diff
@@ -171,4 +173,24 @@ export class ProfileService {
       }
     }).catch(error => {})
   }
+
+  async prefillData(requestDetails: any,entityNames:any,formData:any,showAddOption:any=true) {
```
Comment (Cafnanc) [inline src/app/core/services/profile/profile.service.ts:177]: why are we doing prefill in profile service

File: src/app/core/services/profile/profile.service.ts [hunk 5]
```diff
@@ -171,4 +173,24 @@ export class ProfileService {
...
+    }
+    for (let i = 0; i < formData.controls.length; i++) {
+      if(formData.controls[i].type == 'chip'){
+        formData.controls[i].meta.showAddOption = showAddOption;
+      }
+      formData.controls[i].value = existingData[formData.controls[i].name] ? existingData[formData.controls[i].name] : '';
+      formData.controls[i].options = _.unionBy(
+        formData.controls[i].options,
+        formData.controls[i].value,
+        'value'
```
Comment (Cafnanc) [inline src/app/core/services/profile/profile.service.ts:190]: Also nothing is getting returned after prefill

File: src/app/pages/tabs/profile/profile.page.ts [hunk 6]
```diff
@@ -40,10 +40,18 @@ export class ProfilePage implements OnInit {
...
+        title: 'LANGUAGES',
+        key: 'languages',
+      },
       {
         title: "EMAIL_ID",
         key: "emailId"
       },
+      {
+        title: "ORGANIZATION",
+        key: "organizationName"
```
Comment (Cafnanc) [inline src/app/pages/tabs/profile/profile.page.ts:53]: u can change this back to organization

File: src/app/core/services/profile/profile.service.ts (hunk 4 above)
Comment (GSuma21) [reply src/app/core/services/profile/profile.service.ts:177]: we are prefilling profile data so i kept that function in profile service

File: src/app/core/services/profile/profile.service.ts (hunk 5 above)
Comment (GSuma21) [reply src/app/core/services/profile/profile.service.ts:190]: no return required here form will update directly from this page

File: src/app/pages/tabs/profile/profile.page.ts (hunk 6 above)
Comment (GSuma21) [reply src/app/pages/tabs/profile/profile.page.ts:53]: name conflicts in data so i kept the same ...

**Pull request merged**